src/main/java/com/project/saw/event/EventService.java

```diff
@@ -0,0 +1,92 @@
+package com.project.saw.event;
+
+
+import com.project.saw.dto.event.CreateEventRequest;
+import com.project.saw.dto.event.UpdateEventRequest;
+import com.project.saw.dto.event.UpdateEventResponse;
+import com.project.saw.exception.DuplicateException;
+import jakarta.persistence.EntityNotFoundException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.Comparator;
+import java.util.List;
+
+import java.util.Optional;
+
+
+@Slf4j
+@Service
+public class EventService {
+
+    private final EventRepository eventRepository;
+    private static final String DUPLICATE_EVENT_ERROR_MESSAGE = "This event %s already exist";
+
+    public EventService(EventRepository eventRepository) {
+        this.eventRepository = eventRepository;
+    }
+
+    public List<EventEntity> getEventList() {
+        return eventRepository.findAll().stream()
+                .sorted(Comparator.comparing(EventEntity::getStartingDate))
```

Comment:
sortowanie będzie bardziej wydajne na poziomie query to bazy danych (w repozytorium)